scheduler/__init__.py

```
@@ -0,0 +1,27 @@
+
+Multi-agent safety: every job carries an ``agent_name`` (defaulted
+from ``AGENT_NAME`` env / config) so that two protoAgent instances
+sharing one storage path or one Workstacean install can't accidentally
+fire each other's scheduled prompts.
+"""
+
+from scheduler.interface import Job, SchedulerBackend
+from scheduler.local import LocalScheduler
+from scheduler.workstacean import WorkstaceanScheduler
+
+__all__ = ["Job", "SchedulerBackend", "LocalScheduler", "WorkstaceanScheduler"]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)